UnityClient/Assets/Scripts/Components/UI/Escape/EscapeWindowController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class EscapeWindowController : DraggableUIWindow, IEscapeWindowController {
+
+    [SerializeField]
+    private GameObject ResurrectButton;
+    
+    [SerializeField]
+    private GameObject ReturnToSavePointButton;
+
+    private EntityControl EntityControl;
+    private Toggle CurrentToggle;
+
+    private void Awake() {
+        EntityControl = FindObjectOfType<EntityControl>();
+    }
+
+    public void Resurrect() {
+    }
+
+    public void ToggleSoundUI() {
+    }
+
+    public void ToggleGraphicUI() {
+    }
+
+    public void ReturnToSavePoint() {
+        new CZ.RESTART(0).Send();
+        DisableReturnToSavePoint();
+        Hide();
+    }
+
+    public void ReturnToCharSelection() {
+        new CZ.RESTART(1).Send();
```

Comment:
Maybe put these numbers in a const inside the class?
`public const int RETURN_TO_SAVE_POINT = 0`... etc